src/runtime/vortex/runtime/onnx/onnxruntime.py

```diff
@@ -1,11 +1,9 @@
 import numpy as np
```

Comment:
dont think this file needs to be changed, but okay

Reply:
it is, because if I only install requirements for torchscript it will throw an error as the onnx is not installed and the helper file imported onnx

Reply:
oh yeah i see